schema.graphql

```diff
@@ -1,54 +1,58 @@
-type TokenStaked @entity {
+type Deposit @entity {
   id: ID!
   by: Account!
   amount: BigInt!
+  pool: Pool!
+  txHash: Bytes!
```

Comment:
Why include txhash?

Reply:
I had a txHash incase we ever wanted to look up an event but in a more entity based model we don't need this. Will remove